src/launcher/java/org/truffleruby/launcher/Launcher.java

```diff
@@ -61,7 +62,7 @@
 
     public static final boolean IS_NATIVE = Boolean.getBoolean("com.oracle.graalvm.isaot")
             || Boolean.getBoolean("com.oracle.truffle.aot");
-    public static final String GRAALVM_VERSION = System.getProperty("graalvm.version", "unknown version");
+    public static final String GRAALVM_VERSION = Engine.create().getVersion();
```

Comment:
Should the Engine be disposed?

Reply:
Afaik, no. GraalVM doesn't do it for its temporary engine either.

Reply:
This should probably be `Engine.getVersion()` (a static method), it doesn't make much sense to build an Engine just to get the version. Could you file a feature request for it?

Reply:
I've made a todo and I'll address this later after this PR.

Reply:
OK.